pkg/console/controllers/clidownloads/controller.go

```diff
@@ -201,7 +201,7 @@ odo abstracts away complex Kubernetes and OpenShift concepts, thus allowing deve
 			DisplayName: "odo - Developer-focused CLI for OpenShift",
 			Links: []v1.Link{
 				{
-					Href: "https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/",
+					Href: "https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest",
```

Comment:
This will redirect to `https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/`, so it does work.  We just need to decide if we need to support the trailling `/` and relax the `regex`.

Reply:
already submitted the [PR to fix the validation](https://github.com/openshift/api/pull/485) ... Will probably close this one since the updated link will redirect to the one with the trailing space anyway... @benjaminapetersen thoughts ?

Reply:
Agree!